source/wfs/docker.rst

```diff
@@ -1,8 +1,456 @@
 .. _wfs_docker_chapter:
 
 ###############################################################################
-Docker
+Web-Feature-Service using Docker
 ###############################################################################
 
-Currently available here: `TUM-GIS 3DCityDB WFS Docker
-<https://github.com/tum-gis/3dcitydb-wfs-docker>`_
\ No newline at end of file
+The 3DCityDB Web-Feature-Service (WFS) Docker images expose the capabilities
```

Comment:
We don't use hyphens for "Web-Feature-Service" in the WFS chapter. I think we should be consistent.